--- config-sample.py	2014-07-23 21:24:31.762730111 +0000
+++ config.py	2014-08-27 01:38:34.466730111 +0000
@@ -10,7 +10,7 @@
 DEBUG = True
 
 # Unique name of your site, e.g. "kirsle.net"
-SITE_NAME = "example.com"
+SITE_NAME = "kirsle.net"
 
 # Path to your site's HTML root. Whenever Rophako tries to render a template,
 # it will check in your site's root for the template first before defaulting to
@@ -18,14 +18,14 @@
 # Rophako pages, e.g. for account, blog, photo albums and so on, have templates
 # in the default site. You can override those templates by creating files with
 # the same paths in your site's HTML folder.
-SITE_ROOT = os.path.join(_basedir, "site", "www")
+SITE_ROOT = "/home/kirsle/git/kirsle.net/www"
 
 # E-mail addresses for site notifications (i.e. new comments).
-NOTIFY_ADDRESS = ["root@localhost"]
+NOTIFY_ADDRESS = ["root@kirsle.net"]
 
 # Set this to true if you want your app to force use of SSL. This will also turn
 # on Flask's secure-only session cookies.
-FORCE_SSL = False
+FORCE_SSL = True
 
 # Secret key used for session cookie signing. Make this long and hard to guess.
 #
@@ -50,14 +50,14 @@
 DB_ROOT      = "db"
 REDIS_HOST   = "localhost"
 REDIS_PORT   = 6379
-REDIS_DB     = 0
+REDIS_DB     = 1
 REDIS_PREFIX = "rophako:"
 
 # Mail settings
 MAIL_METHOD = "smtp" # or "sendmail", not yet implemented
 MAIL_SERVER = "localhost"
 MAIL_PORT   = 25
-MAIL_SENDER = "Rophako CMS <no-reply@rophako.kirsle.net>"
+MAIL_SENDER = "Kirsle.net <no-reply@kirsle.net>"
 
 # Emoticon theme used for blog posts and comments. Should exist at the URL
 # "/static/smileys" from your document root, and have a file named
@@ -97,7 +97,7 @@
 # The path to where uploaded photos will be stored.
 # The PRIVATE path is from the perspective of the server file system.
 # The PUBLIC path is from the perspective of the web browser via HTTP.
-PHOTO_ROOT_PRIVATE = os.path.join(_basedir, "site", "www", "static", "photos")
+PHOTO_ROOT_PRIVATE = "/home/kirsle/www/static/photos"
 PHOTO_ROOT_PUBLIC  = "/static/photos"
 
 PHOTO_DEFAULT_ALBUM = "My Photos" # Default/fallback album name.
@@ -135,3 +135,6 @@
 load_plugin("rophako.modules.comment")
 load_plugin("rophako.modules.emoticons")
 load_plugin("rophako.modules.contact")
\ No newline at end of file
+
+# Kirsle legacy endpoints
+load_plugin("kirsle_legacy", as_blueprint=False)
